{"name": "test_customer_signIn[edge]", "status": "passed", "attachments": [{"name": " Test Customer sign in is failed ", "source": "abcc4745-5a3f-4c0a-8fc9-dbcef5b9834b-attachment.png", "type": "image/png"}, {"name": "log", "source": "58bd1f8a-b518-43cc-9831-9b4a12bbd2ef-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "init_driver", "value": "'edge'"}], "start": 1646754193213, "stop": 1646754208305, "uuid": "e52a8b47-01b2-4403-92a3-14464a5a954e", "historyId": "c585e072c0210645023e2a96c76ec875", "testCaseId": "61795a803f78060dfbefbeae26fea2b8", "fullName": "test_customer_signIn.Test_customer_signIn#test_customer_signIn", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "sanity"}, {"name": "suite", "value": "test_customer_signIn"}, {"name": "subSuite", "value": "Test_customer_signIn"}, {"name": "host", "value": "LAPTOP-ESSL3DDB"}, {"name": "thread", "value": "16316-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_customer_signIn"}]}